src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -311,27 +294,60 @@ private List<EntryEditorTab> createTabs() {
                 stateManager,
                 bibEntryTypesManager,
                 keyBindingRepository);
-        entryEditorTabs.add(sourceTab);
-        entryEditorTabs.add(new LatexCitationsTab(databaseContext, preferencesService, dialogService, directoryMonitorManager));
-        entryEditorTabs.add(new FulltextSearchResultsTab(stateManager, preferencesService, dialogService, taskExecutor));
+        tabs.add(sourceTab);
+        tabs.add(new LatexCitationsTab(databaseContext, preferencesService, dialogService, directoryMonitorManager));
+        tabs.add(new FulltextSearchResultsTab(stateManager, preferencesService, dialogService, taskExecutor));
 
-        return entryEditorTabs;
+        return new CreateTabsResult(tabs, tabs.stream().filter(OffersPreview.class::isInstance).map(OffersPreview.class::cast).toList());
```

Comment:
I don't like the result pattern. It always smells like there is an architectural flaw.

Reply:
But i'm fine with it, looks like its just an iteration step.

Reply:
Fixed in [`365c108` (#11379)](https://github.com/JabRef/jabref/pull/11379/commits/365c1087d5824e22c18d1bc639fbd1071411fa22)